src/build/config/config.yaml

```diff
@@ -2220,6 +2221,77 @@ option:
     command: repo-type
     depend: repo-s3-bucket
 
+  repo-sftp-account:
+    section: global
+    type: string
+    group: repo
+    command: repo-type
+    depend:
+      option: repo-type
+      list:
+        - sftp
+
+  repo-sftp-host:
```

Comment:
I'll remove this.  repo-host should be used. I initially segregated out all the sftp params.

Reply:
Are we sure that this makes sense..?  GCS, s3, Azure all have their own "endpoint" values that they're configured with and those seem to be a lot closer to how SFTP works in terms of connecting to a remote system and issuing file operations against it.

Reply:
Per discussion, sounds like we want to keep the sftp-specific configuration parameters as it's more in-line with how s3, gcs, azure work and sftp is more similar to those.